src/app/backend/resource/pod/poddetail.go

```diff
@@ -15,17 +15,23 @@
 package pod
 
 import (
+	"encoding/json"
 	"log"
 
 	"k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/apis/batch"
 	k8sClient "k8s.io/kubernetes/pkg/client/unversioned"
 
 	"github.com/kubernetes/dashboard/src/app/backend/client"
 	"github.com/kubernetes/dashboard/src/app/backend/resource/common"
 	"github.com/kubernetes/dashboard/src/app/backend/resource/dataselect"
+	"github.com/kubernetes/dashboard/src/app/backend/resource/job/joblist"
 	"github.com/kubernetes/dashboard/src/app/backend/resource/metric"
 )
 
+const (
+	CreatedByAnnotation = "kubernetes.io/created-by"
```

Comment:
@bryk I think it's good idea to update Go dependencies, because string like this are already added into core code: 

``` go
// CreatedByAnnotation represents the key used to store the spec(json)
// used to create the resource.
CreatedByAnnotation = "kubernetes.io/created-by"
```

It's likely, that we'll find more useful code.